pkg/model/logstore/logstore.go

```diff
@@ -679,10 +684,27 @@ func (s *LogStore) toLogLines(options logOptions) []LogLine {
 			}
 		}
 
+		if buildEvent == "init" {
+			// Estimate width of a "normal" Terminal
+			const lineLength = 80
+
+			// If the text exceeds lineLength, still add a short border
+			const minBorderLength = 5
+			textPadding := 2
+			borderDashCount := lineLength - len(segment.Text) - textPadding
+			if borderDashCount < minBorderLength {
+				borderDashCount = minBorderLength
+			}
+
+			border := "\n" + strings.Repeat("═", borderDashCount) + "╡ "
```

Comment:
This adds a newline character to the middle of a LogLine, which is going to have weird effects.

This probably needs to be

```
sb.WriteString("\n")
maybePushLine()
sb.WriteString(border)

Reply:
Discussed this IRL with @nicks and decided to move all these changes into a separate PR, since we need to be a little more thoughtful about how we manipulate strings in logstore.